app/screens/network/Network.js

```diff
@@ -0,0 +1,207 @@
+  status: Object,
+  nodeIndicator: Object,
+  history: RouterHistory
+};
+
+class Network extends Component<Props, State> {
+  startUpDelay = 5; // eslint-disable-line react/sort-comp
+
+  noPeersCounter = 0; // eslint-disable-line react/sort-comp
+
+  color;
+
+  render() {
+    const { node, status, nodeIndicator } = this.props;
+
+    const networkName = 'TweedleDee 0.1.0';
+
+    return (
+      <WrapperWith2SideBars width={1000} height={500} header="NETWORK" headerIcon={network} subHeader={networkName}>
+        <Container>
+          <Message color={nodeIndicator.color}>{nodeIndicator.message}</Message>
+          <DetailsWrap>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Age</DetailsText>
+                <Tooltip width="250" text="tooltip age" />
+              </DetailsTextWrap>
+              <GrayText>{getFormattedTimestamp(node.genesisTime)}</GrayText>
+            </DetailsRow>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Status</DetailsText>
+                <Tooltip width="250" text="tooltip Status" />
+              </DetailsTextWrap>
+              <GrayText>{this.renderSyncingStatus(status)}</GrayText>
+            </DetailsRow>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Current Layer</DetailsText>
+                <Tooltip width="250" text="tooltip Current Layer" />
+              </DetailsTextWrap>
+              <GrayText>{node.status.currentLayer}</GrayText>
+            </DetailsRow>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Verified Layer</DetailsText>
+                <Tooltip width="250" text="tooltip Verified Layer" />
+              </DetailsTextWrap>
+              <GrayText>{node.status.verifiedLayer}</GrayText>
+            </DetailsRow>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Connection Type</DetailsText>
+                <Tooltip width="250" text="tooltip Connection Type" />
+              </DetailsTextWrap>
+              <GrayText>Managed p2p node</GrayText>
+            </DetailsRow>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Connected neighbors</DetailsText>
+                <Tooltip width="250" text="tooltip Connected neighbors" />
+              </DetailsTextWrap>
+              <GrayText>8</GrayText>
+            </DetailsRow>
+          </DetailsWrap>
+          <FooterWrap>
+            <Link onClick={this.navigateToChangeNetwork} text="CHANGE NETWORK" />
+            <Tooltip width="250" text="tooltip CHANGE NETWORK" />
+            <Link onClick={this.openLogFile} text="BROWSE LOG FILE" style={{ marginLeft: '50px' }} />
+            <Tooltip width="250" text="tooltip BROWSE LOG FILE" />
+          </FooterWrap>
+        </Container>
+      </WrapperWith2SideBars>
+    );
+  }
+
+  renderSyncingStatus = (status) => {
```

Comment:
no need to pass as arg a prop 'status', take it from 'this.props'

Reply:
fix